tasks/aide.yml

```
@@ -70,14 +70,28 @@
     line: "{{ item }} p+i+n+u+g+s+b+acl+xattrs+sha512"
     mode: "0640"
     create: false
+    validate: aide --config-check --config=%s
   with_items:
     - /usr/sbin/auditctl
     - /usr/sbin/auditd
     - /usr/sbin/ausearch
     - /usr/sbin/aureport
     - /usr/sbin/autrace
     - /usr/sbin/augenrules
-    - /usr/sbin/audispd
+    # - /usr/sbin/audispd # This is not required for CIS Benchmarks, so we won't be installing audispd
```

Remove unneeded capitalization in a comment:
```suggestion
    # - /usr/sbin/audispd # This is not required for CIS benchmarks, so we won't be installing audispd
```